src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs

```diff
@@ -201,10 +201,13 @@ static void PrintSettings(IEnumerable<KeyValuePair<string, object>> settings)
                     var maxTime = runDescriptor.Settings.TestExecutionTimeout ?? TimeSpan.FromSeconds(90);
                     while (!done() && !cts.Token.IsCancellationRequested)
                     {
-                        if (DateTime.UtcNow - startTime > maxTime)
+                        if (!Debugger.IsAttached)
```

Comment:
Looking at the `Debugger` source it looks like the property has a getter only. I can't see anywhere it's being set though so I'm not 100% sure it's "cached" or not